### ISSUE TITLE
World Server Crash

### ISSUE THREAD

**Issue by slash-design** [jasongdove/LegionCore-7.3.5#97]
can't log into world...

crash..

locale ?

[_worldserver.exe_[27-10_23-4-25].txt](https://github.com/user-attachments/files/17535585/_worldserver.exe_.27-10_23-4-25.txt)

**Comment by slash-design** [comment]
6bccb1b113fab9c6603f31b80de61e57d6218db3
this is the bad boy :-)

**Comment by jasongdove** [comment]
Yeah, I set a breakpoint on that line because I wasn't sure if I ported that correctly but it wasn't ever hit with my locale.

**Comment by jasongdove** [comment]
Which locale do you test with?

**Comment by slash-design** [comment]
deDe

I will try it this way later ....

case FT_STRING:
                    buffer << (*reinterpret_cast<LocalizedString const*>(entry))[locale];
                    entry += sizeof(LocalizedString);
                    break;

**Comment by Etoyspeh** [comment]
@jasongdove Thank you, it's working now.
